src/fpu/fpu_instructions.h

```diff
@@ -107,16 +111,24 @@ static double FROUND(double in){
 	case ROUND_Up: return std::ceil(in);
 	case ROUND_Chop: 
 	{
+		switch (fpu.cw & PrecisionModeMask) {
+		// This case properly handles "in" values beyond the precision
+		// of a float, e.g. 5.99999999367, by correctly rounding to 6.0
+		// via the cast, then performing the truncation,
+		// instead of returning 5.0. Quake, for example, does many
+		// calculations in single precision mode.
```

Comment:
Minor: reformat comment block

Reply:
ah, ok, had to put it back to a single-line comment and let the script do it over again.